src/server/render.jsx

```diff
@@ -43,6 +43,11 @@ function loadWebpackAssets() {
 }
 
 
+function trimEndSlash(value) {
```

Comment:
Unless I am mistaken, this regexp will actually remove _all_ trailing slashes, not only the last one.
```suggestion
function trimTrailingSlashes(value) {
```